base/applications/rapps/lang/he-IL.rc

```diff
@@ -204,20 +204,20 @@ BEGIN
     IDS_APPLICATIONS "יישומים"
     IDS_CHOOSE_FOLDER_TEXT "בחר תקייה שתשמור הורדות:"
     IDS_CHOOSE_FOLDER_ERROR "התקייה שהזנת לא קיימת, ליצור אותה?"
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "כתובת האתר שציינת אינה חוקית או שאינה נתמכת. בבקשה תתקן זאת!"
     IDS_APP_REG_REMOVE "Are you sure you want to delete the data on the installed program from the registry?"
```

Comment:
Is there any specific reason why you decided to not translate all remaining english strings in this file?

Reply:
I didn't quite know how to translate the word "Registry" into Hebrew
Might have to look into how windows calls it

Reply:
Note that there are other untranslated strings in this file, as well as in the fontview one too.